### PR TITLE
Update storage_volumes.md

### PR DESCRIPTION
**Pull request by piperdeck**
Added a point mentioning the restriction that storage volumes can't be attached to running virtual machines. This is mentioned by Stephane Graber in this youtube tutorial [1] and has been confirmed experimentally (at least for a volume with a "dir" backend)

[1] https://youtu.be/dvQ111pbqtk?si=VNRiMlpZ_h6TRzn8&t=410

Signed-off-by: Piper Deck <piper.deck@canonical.com>

### CODE REVIEW

**Comment by tomponline** [comment]
Hi @piperdeck your PR got me thinking and resulted in this PR as well https://github.com/canonical/lxd/pull/12230 :)

Anyway, the end result is that I think we need to clarify the statement you've added here to say that:

> - Filesystem storage volumes can't be attached to virtual machines while they're running.

As it is possible to attach block devices, block custom volumes or iso files to running VMs.

**Comment by piperdeck** [comment]
Hi @tomponline, thanks for that correction! I've added that detail to the PR. Please let me know if there's anything else I need to change.

File: doc/howto/storage_volumes.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ The following restrictions apply:
 - Custom storage volumes of {ref}`content type <storage-content-types>` `block` or `iso` cannot be attached to containers, but only to virtual machines.
 - To avoid data corruption, storage volumes of {ref}`content type <storage-content-types>` `block` should never be attached to more than one virtual machine at a time.
 - Storage volumes of {ref}`content type <storage-content-types>` `iso` are always read-only, and can therefore be attached to more than one virtual machine at a time without corrupting data.
+- Filesystem storage volumes can't be attached to virtual machines while they're running.
```
Comment (tomponline) [inline doc/howto/storage_volumes.md:55]: Sorry my suggestion of "Filesystem" here has fallen foul of our spell checker which wants "File system".

Comment (piperdeck) [reply doc/howto/storage_volumes.md]: Done!

**Pull request merged**